config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/", to: "welcome#index"
+
+  get "/ingredients", to: "ingredients#index"
+  get "/recipes/:id", to: "recipes#show"
+  # post "/recipes/:id", to: "recipe_ingredients#create"
+  post "/recipes/:id", to: "recipe_ingredients#create"
```

Comment:
see note in the recipes show.html.erb to see how you could make this route more RESTful.